add config management (#1)
Alex-Izquierdo/ansible-events-qa: README.md

```
./utils/api-client-generator/generate.sh
```

## Configuration

> Configuration is defined in `ansible_events_qa/conf/settings.yaml`, is managed by [dynaconf library](https://dynaconf.readthedocs.io/).

Reviewer: ```suggestion
Configuration is defined in `ansible_events_qa/conf/settings.yaml` and is managed by [dynaconf library](https://dynaconf.readthedocs.io/).
```

Supports a `.env` file in the root of the project which can have defined environment variables to override default settings.
You can override it with `AEQE_{keyname}={keyvalue}`
The configuration supports multiple environments, you can define it with the environment variable `AEQE_ENV_{environment}`

### encrypted values

> The values can have encrypted values that are automatically decrypted. You must define the environment variable `AEQE_FERNET_PASSWORD` if you have some encrypted value. The encrypted values must follow the pattern `key: "!fernat:[encrypted data]"`

Reviewer: ```suggestion
You can create encrypted values that will be automatically decrypted. Before you can encrypt a value you must first specify the fernet password by defining the environment variable `AEQE_FERNET_PASSWORD`. If you do not have the fernet password, please contact a team member to acquire it.

You can then use the encrypted value by specifying it with the following pattern: `key: "!fernat:[encrypted data]"`
```

> The values can have encrypted values that are automatically decrypted. You must to define the environment variable `AEQE_FERNET_PASSWORD` if you have some encrypted value. The encrypted values must follow the pattern `key: "!fernat:[encrypted data]"`

Reviewer: You must to define > You must define

Author: thanks

> The encryption uses `cryptography.fernet` which is an easy and reliable simmetric encryption based on AES cipher.

Reviewer: simmetric > symmetric

You can use `aeqe` command to encrypt or decrypt values:

```shell
aeqe --encrypt "password"
# gAAAAABjEPXUFt5Oc-ivGsv6SZ9goAp8X_WIIBRJvzvIm7n8AGn18BL9tsAEGW6P2BUN2GjOXyBG711dEGg2s-qOJVAvjR0m4g==
aeqe --decrypt "gAAAAABjEPXUFt5Oc-ivGsv6SZ9goAp8X_WIIBRJvzvIm7n8AGn18BL9tsAEGW6P2BUN2GjOXyBG711dEGg2s-qOJVAvjR0m4g=="
# password
```

> If you don't have the fernet password, contact with some team member to adquire it.

Reviewer: ```suggestion
```
Relocated to the "encrypted values" section.
